release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -80,6 +80,50 @@ For more information, see xref:../backup_and_restore/hibernating-cluster.adoc#hi
 [id="ocp-release-notes-extensions_{context}"]
 === Extensions ({olmv1})
 
+{olmv0-first-caps} has been included with {product-title} 4 since its initial release. {product-title} {product-version} introduces the next-generation iteration of cluster extension lifecycling: {olmv1-first}. Previously available as a Technology Preview feature only, this updated framework evolves many of the concepts that have been part of previous versions of {olmv0} and adds new capabilities.
```

Comment:
From the extensions overview page: https://docs.openshift.com/container-platform/4.18/extensions/index.html